src/modules/dashboard/components/Dialogs/CreatePaymentDialog/CreatePaymentDialogForm.tsx

```diff
@@ -501,10 +501,9 @@ const CreatePaymentDialogForm = ({
               dataTest="paymentAmountInput"
             />
             {networkFeeInverse &&
+              customAmountError === undefined &&
               values.amount &&
-              values.amount !== '0' &&
-              values.amount !== '0.' &&
-              values.amount !== '.' && (
+              Number(values.amount) > 0 && (
```

Comment:
I like this change. Just a heads up that Armando is implementing something similar in #3535

So maybe you guys should coordinate to avoid merge conflicts

Reply:
Thanks for the heads up Raul. I already talked to Armando and he likes my changes more so I am going to go ahead and merge them.

Reply:
> I like this change. Just a heads up that Armando is implementing something similar in #3535
> 
> So maybe you guys should coordinate to avoid merge conflicts

@rdig Not sure #3535 is the one.....